blockchains/erc20/lib/fetch_events.js

```diff
@@ -151,13 +151,13 @@ async function decodeWETHDepositEvent(web3Wrapper, event, timestampsCache) {
 /**Withdrawal(address,uint256)
  * Only for WETH
  **/
-async function decodeWETHWithdrawalEvent(web3Wrapper, event, timestampsCache) {
+function decodeWETHWithdrawalEvent(web3Wrapper, event, timestampsCache) {
   if (event['address'].toLowerCase() !== WETH_contract
     || event['topics'].length !== 2) {
     return null;
   }
 
-  let result = await decodeEventBasicInfo(web3Wrapper, event, timestampsCache);
+  const result = decodeEventBasicInfo(event, timestampsCache);
```

Comment:
The `timestampsCache` would already contain the data we need (see below). We do not do an async call to fill it anymore at this point. As result we do not 'await'.